packages/@tailwindcss-upgrade/src/index.ts

```diff
@@ -229,11 +228,22 @@ async function run() {
     }
 
     info('Updating dependencies…')
-    try {
-      // Upgrade Tailwind CSS
-      await pkg(base).add(['tailwindcss@latest'])
-      success(`Updated package: ${highlight('tailwindcss')}`, { prefix: '↳ ' })
-    } catch {}
+    for (let dependency of [
+      'tailwindcss',
+      '@tailwindcss/cli',
+      '@tailwindcss/postcss',
```

Comment:
The PostCSS config is migrated later, which means that on v3 projects at the time of running this will result in a no-op.